src/trix/config/text_attributes.coffee

```diff
@@ -3,8 +3,10 @@ Trix.config.textAttributes =
     tagName: "strong"
     inheritable: true
     parser: (element) ->
+      return false if /H\d$/.test(element.tagName) or element.tagName is "BR"
```

Comment:
This should be removed. It was fixed in master with 98931a15c17f29c2cfec20cfeda228e8124bb8ed